feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.ts

```diff
@@ -57,4 +75,99 @@ export class ConfiguratorCartEntryBundleInfoService {
     }
     return preparedLabel;
   }
+
+  protected processConfigurationInfos(
+    configurationInfos: ConfigurationInfo[]
+  ): LineItem[] {
+    const lineItemMap: Map<number, LineItem> = new Map();
+    configurationInfos.forEach((configurationInfo) =>
+      this.processConfigurationInfoEntry(lineItemMap, configurationInfo)
+    );
+    // sort
+    const lineItemMapSorted: Map<number, LineItem> = new Map<number, LineItem>(
+      Array.from(lineItemMap).sort((a, b) => {
+        return a[0] - b[0];
+      })
+    );
+    // convert to array
+    const lineItems: LineItem[] = Array.from(lineItemMapSorted.values());
+    return lineItems;
+  }
+
+  protected processConfigurationInfoEntry(
+    lineItemMap: Map<number, LineItem>,
+    configurationInfo: ConfigurationInfo
+  ): void {
+    if (configurationInfo.configurationLabel) {
+      const configurationInfoSplitted: string[] = configurationInfo.configurationLabel.split(
+        ConfigurationInfoSpecialFields.LINEITEM_DELIMITER
+      );
+      if (
+        configurationInfoSplitted[0] === ConfigurationInfoSpecialFields.LINEITEM
+      ) {
+        const configurationInfoValue: string = configurationInfo.configurationValue
+          ? configurationInfo.configurationValue
+          : '';
+        this.addLineItemData(
+          lineItemMap,
+          configurationInfoSplitted,
+          configurationInfoValue
+        );
+      }
+    }
+  }
+
+  protected addLineItemData(
+    lineItemMap: Map<number, LineItem>,
+    configurationInfoSplitted: string[],
```

Comment:
Nitpick: Please consider to rename (split past tense is 'split')

Reply:
done